packages/core/src/amazonqFeatureDev/client/featureDev.ts

```diff
@@ -184,6 +185,13 @@ export class FeatureDevClient {
                     e.name === 'ServiceQuotaExceededException'
                 ) {
                     throw new PlanIterationLimitError()
+                } else if (
+                    e.name === 'ValidationException' &&
+                    e.message.includes('repo size is exceeding the limits')
```

Comment:
it looks like this same message is being used twice in two different places. Can we extract it somewhere?

Same with `zipped file is corrupted`

Reply:
sure good idea